{"uid":"99323ae0da684be3","name":"test_smoke","fullName":"tests.integration.test_smoke#test_smoke","historyId":"78ffebf457376473b3476ee0f7a454c9","time":{"start":1752283593241,"stop":1752284806224,"duration":1212983},"description":"Deploy bundle with apps and test various component integrations.","descriptionHtml":"<p>Deploy bundle with apps and test various component integrations.</p>\n","status":"broken","statusMessage":"TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 520, in wait_for\n    return await fut\n           ^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 665, in sleep\n    return await future\n           ^^^^^^^^^^^^\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/tests/integration/test_smoke.py\", line 68, in test_smoke\n    await ensure_statuses(ops_test)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/tests/integration/test_smoke.py\", line 40, in ensure_statuses\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1046, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 519, in wait_for\n    async with timeouts.timeout(timeout):\n  File \"/usr/lib/python3.12/asyncio/timeouts.py\", line 115, in __aexit__\n    raise TimeoutError from exc_val\nTimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1752283593004,"stop":1752283593004,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1752283593004,"stop":1752283593004,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1752283593004,"stop":1752283593240,"duration":236},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1752283593240,"stop":1752283593240,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy bundle with apps and test various component integrations.","status":"broken","statusMessage":"TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 520, in wait_for\n    return await fut\n           ^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 665, in sleep\n    return await future\n           ^^^^^^^^^^^^\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/tests/integration/test_smoke.py\", line 68, in test_smoke\n    await ensure_statuses(ops_test)\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/tests/integration/test_smoke.py\", line 40, in ensure_statuses\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1046, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-k8s-bundle/mysql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.12/asyncio/tasks.py\", line 519, in wait_for\n    async with timeouts.timeout(timeout):\n  File \"/usr/lib/python3.12/asyncio/timeouts.py\", line 115, in __aexit__\n    raise TimeoutError from exc_val\nTimeoutError","steps":[],"attachments":[{"uid":"4ca75cf8ef713c15","name":"log","source":"4ca75cf8ef713c15.txt","type":"text/plain","size":73755}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1752284806272,"stop":1752284806272,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1752284806273,"stop":1752284806915,"duration":642},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1752284806915,"stop":1752284806915,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1752284806915,"stop":1752284806915,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1752284806915,"stop":1752284806915,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"pkrvmq0rgcvqdmg"},{"name":"thread","value":"16270-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d02214d48921e382","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":34,"broken":136,"skipped":0,"passed":164,"unknown":8,"total":342},"items":[{"uid":"3e534f89c9c14f64","reportUrl":"../1942//#testresult/3e534f89c9c14f64","status":"broken","statusDetails":"TimeoutError","time":{"start":1752197175543,"stop":1752198389457,"duration":1213914}},{"uid":"814a263ac1bcc9b5","reportUrl":"../1941//#testresult/814a263ac1bcc9b5","status":"broken","statusDetails":"TimeoutError","time":{"start":1752110774036,"stop":1752112009966,"duration":1235930}},{"uid":"5f9a44402dce594a","reportUrl":"../1940//#testresult/5f9a44402dce594a","status":"broken","statusDetails":"TimeoutError","time":{"start":1752024351484,"stop":1752025568288,"duration":1216804}},{"uid":"de50ac113ccf0581","reportUrl":"../1938//#testresult/de50ac113ccf0581","status":"broken","statusDetails":"TimeoutError","time":{"start":1751937939323,"stop":1751939155960,"duration":1216637}},{"uid":"53d54ab759c091c6","reportUrl":"../1937//#testresult/53d54ab759c091c6","status":"broken","statusDetails":"TimeoutError","time":{"start":1751851532591,"stop":1751852746211,"duration":1213620}},{"uid":"ac69e8844ac3f393","reportUrl":"../1936//#testresult/ac69e8844ac3f393","status":"broken","statusDetails":"TimeoutError","time":{"start":1751765153958,"stop":1751766369462,"duration":1215504}},{"uid":"d62b9e6ffb758bc8","reportUrl":"../1935//#testresult/d62b9e6ffb758bc8","status":"broken","statusDetails":"TimeoutError","time":{"start":1751678742302,"stop":1751679956283,"duration":1213981}},{"uid":"618853ff953a1367","reportUrl":"../1934//#testresult/618853ff953a1367","status":"broken","statusDetails":"TimeoutError","time":{"start":1751592381067,"stop":1751593599498,"duration":1218431}},{"uid":"843e67082cfe602c","reportUrl":"../1933//#testresult/843e67082cfe602c","status":"broken","statusDetails":"TimeoutError","time":{"start":1751505909137,"stop":1751507123537,"duration":1214400}},{"uid":"72c37528f151548f","reportUrl":"../1931//#testresult/72c37528f151548f","status":"broken","statusDetails":"TimeoutError","time":{"start":1751419525865,"stop":1751420740154,"duration":1214289}},{"uid":"c27f69959d691774","reportUrl":"../1928//#testresult/c27f69959d691774","status":"broken","statusDetails":"TimeoutError","time":{"start":1751333190072,"stop":1751334404462,"duration":1214390}},{"uid":"a714d6973cd0cc1b","reportUrl":"../1926//#testresult/a714d6973cd0cc1b","status":"broken","statusDetails":"TimeoutError","time":{"start":1751246733846,"stop":1751247948206,"duration":1214360}},{"uid":"21f6189ea10c8a13","reportUrl":"../1924//#testresult/21f6189ea10c8a13","status":"broken","statusDetails":"TimeoutError","time":{"start":1751160392500,"stop":1751161606841,"duration":1214341}},{"uid":"b0a5d1e90d785012","reportUrl":"../1922//#testresult/b0a5d1e90d785012","status":"broken","statusDetails":"TimeoutError","time":{"start":1751073931985,"stop":1751075149807,"duration":1217822}},{"uid":"2f62aead85cf15b7","reportUrl":"../1920//#testresult/2f62aead85cf15b7","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent-k8s : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@22.04\n  channel \"1/candidate\": available bases are: ubuntu@22.04\n  channel \"1/beta\": available bases are: ubuntu@22.04\n  channel \"1/edge\": available bases are: ubuntu@22.04\n  channel \"2/edge\": available bases are: ubuntu@24.04","time":{"start":1750987544334,"stop":1750987548298,"duration":3964}},{"uid":"a5a2345b207d5896","reportUrl":"../1918//#testresult/a5a2345b207d5896","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent-k8s : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64\"\navailable releases are:\n  channel \"1/stable\": available bases are: ubuntu@22.04\n  channel \"1/candidate\": available bases are: ubuntu@22.04\n  channel \"1/beta\": available bases are: ubuntu@22.04\n  channel \"1/edge\": available bases are: ubuntu@22.04\n  channel \"2/edge\": available bases are: ubuntu@24.04","time":{"start":1750901206649,"stop":1750901209742,"duration":3093}},{"uid":"a5f01accb8ab02a4","reportUrl":"../1917//#testresult/a5f01accb8ab02a4","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent-k8s : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64\"\navailable releases are:\n  channel \"2/edge\": available bases are: ubuntu@24.04\n  channel \"1/stable\": available bases are: ubuntu@22.04\n  channel \"1/candidate\": available bases are: ubuntu@22.04\n  channel \"1/beta\": available bases are: ubuntu@22.04\n  channel \"1/edge\": available bases are: ubuntu@22.04","time":{"start":1750814800985,"stop":1750814804649,"duration":3664}},{"uid":"5219addf36251ef4","reportUrl":"../1915//#testresult/5219addf36251ef4","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent-k8s : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64\"\navailable releases are:\n  channel \"1/edge\": available bases are: ubuntu@22.04\n  channel \"2/edge\": available bases are: ubuntu@24.04\n  channel \"1/stable\": available bases are: ubuntu@22.04\n  channel \"1/candidate\": available bases are: ubuntu@22.04\n  channel \"1/beta\": available bases are: ubuntu@22.04","time":{"start":1750728330124,"stop":1750728330617,"duration":493}},{"uid":"2b706a920b6dd6e2","reportUrl":"../1914//#testresult/2b706a920b6dd6e2","status":"unknown","time":{}},{"uid":"f632e44f40be6a89","reportUrl":"../1913//#testresult/f632e44f40be6a89","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:grafana-agent-k8s : selecting releases: charm or bundle not found for channel \"latest/edge\", base \"amd64\"\navailable releases are:\n  channel \"1/beta\": available bases are: ubuntu@22.04\n  channel \"1/edge\": available bases are: ubuntu@22.04\n  channel \"2/edge\": available bases are: ubuntu@24.04\n  channel \"1/stable\": available bases are: ubuntu@22.04\n  channel \"1/candidate\": available bases are: ubuntu@22.04","time":{"start":1750555624110,"stop":1750555625002,"duration":892}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"99323ae0da684be3.json","parameterValues":[]}